README.rst

```diff
@@ -460,75 +461,6 @@ Tests execution
 2) Assign the PYTHONPATH environment variable executing "export PYTHONPATH=../.."
 3) Run lettuce_tools with appropriate params (see available ones with the -h option)
```

Comment:
Wow, here make reference to lettuce, I think that it is something to check because I thought that all tests are executed with behave now. Please check it.